packages/react-instantsearch/src/connectors/connectInfiniteHits.test.js

```diff
@@ -41,7 +42,7 @@ describe('connectInfiniteHits', () => {
       const hits2 = [{}, {}, {}, {}, {}, {}];
       const hits3 = [{}, {}, {}, {}, {}, {}, {}, {}];
       const res1 = getProvidedProps(null, null, {
-        results: { hits, page: 0, hitsPerPage: 6, nbPages: 10 },
+        results: { hits, page: 0, hitsPerPage: 6, nbPages: 10, index: 'index' },
```

Comment:
If you're using the `createConnector` API yes, because you access `searchResults` otherwise it shouldn't.